homeassistant/components/becker/cover.py

```diff
@@ -0,0 +1,191 @@
+"""Support for Becker RF covers."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.cover import (
+    SUPPORT_CLOSE,
+    SUPPORT_CLOSE_TILT,
+    SUPPORT_OPEN,
+    SUPPORT_OPEN_TILT,
+    SUPPORT_STOP,
+    CoverEntity,
+)
+from homeassistant.const import (
+    CONF_COVERS,
+    CONF_NAME,
+    CONF_VALUE_TEMPLATE,
+    STATE_CLOSED,
+    STATE_OPEN,
+)
+from homeassistant.exceptions import TemplateError
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.restore_state import RestoreEntity
+
+from .const import (
+    CLOSED_POSITION,
+    CONF_CHANNEL,
+    DEVICE_CLASS,
+    DOMAIN,
+    INTERMEDIATE_POSITION,
+    OPEN_POSITION,
+    VENTILATION_POSITION,
+)
+from .utils import extract_entities, initialise_templates
+
+_LOGGER = logging.getLogger(__name__)
+
+COVER_FEATURES = (
+    SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP | SUPPORT_OPEN_TILT | SUPPORT_CLOSE_TILT
+)
+
+
+_VALID_STATES = [STATE_OPEN, STATE_CLOSED, "true", "false"]
+
+COVER_SCHEMA = vol.Schema(
```

Comment:
This is not used.

Reply:
I think it's used in 
`if state in _VALID_STATES:` (line 177)

Reply:
os tempate support is removed I as well removed the _VALID_STATES